zeebe/snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/SfvChecksumImpl.java

```diff
@@ -135,31 +99,25 @@ public void updateFromFile(final Path filePath) throws IOException {
 
   @Override
   public void updateFromBytes(final String fileName, final byte[] bytes) {
-    combinedChecksum.update(fileName.getBytes(UTF_8));
     final Checksum checksum = new CRC32C();
     checksum.update(bytes);
-    combinedChecksum.update(bytes);
     checksums.put(fileName, checksum.getValue());
   }
 
   @Override
   public void updateFromSfvFile(final String... lines) {
     for (String line : lines) {
       line = line.trim();
+
       if (line.startsWith(";")) {
-        final Matcher matcher = COMBINED_VALUE_PATTERN.matcher(line);
-        if (matcher.find()) {
-          final String hexString = matcher.group(1);
-          final long crc = Long.parseLong(hexString, 16);
-          combinedChecksum = new PreDefinedImmutableChecksum(crc);
-        }
-      } else {
-        final Matcher matcher = FILE_CRC_PATTERN.matcher(line);
-        if (matcher.find()) {
-          final Long crc = Long.parseLong(matcher.group(2), 16);
-          final String fileName = matcher.group(1).trim();
-          checksums.put(fileName, crc);
-        }
+        continue;
```

Comment:
:question: This is for backwards compatibility so that it can still parse the old files. Is that right?

Reply:
Yes it can still parse old files, it will ignore all commented lines (as they were just used to store combined checksum info) and read all normal checksum lines